client/packages/requisitions/src/RequestRequisition/DetailView/IndicatorEdit/IndicatorLineEdit.tsx

```diff
@@ -119,7 +119,8 @@ export const IndicatorLineEdit = ({
   const { store } = useAuthContext();
   const showInfo =
     store?.preferences.useConsumptionAndStockFromCustomersForInternalOrders &&
-    !!currentLine?.customerIndicatorInfo;
+    store?.preferences?.extraFieldsInRequisition;
```

Comment:
Was missing required in specs same for showItemInfo down below

Reply:
Ohh really, so can't use indicators without this preference in mSupply ?

Reply:
Will have to double check